diff -Naur a/luasrc/model/cbi/turboacc.lua b/luasrc/model/cbi/turboacc.lua
--- a/luasrc/model/cbi/turboacc.lua	2022-03-12 17:46:47.068413861 +0000
+++ b/luasrc/model/cbi/turboacc.lua	2022-03-12 17:47:13.100144807 +0000
@@ -43,10 +43,10 @@
 sfe_ipv6:depends("sfe_flow", 1)
 end
 
-if nixio.fs.access("/lib/modules/" .. kernel_version .. "/tcp_bbr.ko") then
-bbr_cca = s:option(Flag, "bbr_cca", translate("BBR CCA"))
+if nixio.fs.access("/lib/modules/" .. kernel_version .. "/tcp_bbr2.ko") then
+bbr_cca = s:option(Flag, "bbr_cca", translate("BBR2 CCA"))
 bbr_cca.default = 0
-bbr_cca.description = translate("Using BBR CCA can improve TCP network performance effectively")
+bbr_cca.description = translate("Using BBR2 CCA can improve TCP network performance effectively")
 end 
 
 if nixio.fs.access("/lib/modules/" .. kernel_version .. "/nft_fullcone.ko") then
diff -Naur a/luasrc/view/turboacc/status.htm b/luasrc/view/turboacc/status.htm
--- a/luasrc/view/turboacc/status.htm	2022-03-12 17:46:47.112105865 +0000
+++ b/luasrc/view/turboacc/status.htm	2022-03-12 17:47:13.100144807 +0000
@@ -3,7 +3,7 @@
 	<table width="100%" cellspacing="10" id="_turboacc_status_table">
 		<tr><td width="33%"><%:Flow Offloading%></td><td id="_fastpath_state"><em><%:Collecting data...%></em></td></tr>
 		<tr><td width="33%"><%:FullCone NAT%></td><td id="_fullconenat_state"><em><%:Collecting data...%></em></td></tr>
-		<tr><td width="33%"><%:BBR CCA%></td><td id="_bbr_state"><em><%:Collecting data...%></em></td></tr>
+		<tr><td width="33%"><%:BBR2 CCA%></td><td id="_bbr_state"><em><%:Collecting data...%></em></td></tr>
 		<tr><td width="33%"><%:DNS Caching%></td><td id="_dnscaching_state"><em><%:Collecting data...%></em></td></tr>
 	</table>
 </fieldset>
diff -Naur a/Makefile b/Makefile
--- a/Makefile	2022-03-12 17:46:46.977057864 +0000
+++ b/Makefile	2022-03-12 17:47:13.152152943 +0000
@@ -13,7 +13,7 @@
 
 LUCI_TITLE:=LuCI support for Flow Offload / Shortcut-FE
 LUCI_DEPENDS:=+pdnsd-alt \
-	+PACKAGE_TURBOACC_INCLUDE_BBR_CCA:kmod-tcp-bbr \
+	+PACKAGE_TURBOACC_INCLUDE_BBR_CCA:kmod-tcp-bbr2 \
 	+PACKAGE_TURBOACC_INCLUDE_DNSFORWARDER:dnsforwarder \
 	+PACKAGE_TURBOACC_INCLUDE_DNSPROXY:dnsproxy \
 	+PACKAGE_TURBOACC_INCLUDE_OFFLOADING:kmod-nft-offload \
diff -Naur a/po/zh_Hans/turboacc.po b/po/zh_Hans/turboacc.po
--- a/po/zh_Hans/turboacc.po	2022-03-12 17:46:47.028693862 +0000
+++ b/po/zh_Hans/turboacc.po	2022-03-12 17:47:13.052137295 +0000
@@ -40,11 +40,11 @@
 msgid "Enable IPv6 Acceleration"
 msgstr "启用 IPv6 加速"
 
-msgid "BBR CCA"
-msgstr "BBR 拥塞控制算法"
+msgid "BBR2 CCA"
+msgstr "BBR2 拥塞控制算法"
 
-msgid "Using BBR CCA can improve TCP network performance effectively"
-msgstr "使用 BBR 拥塞控制算法可以有效提升 TCP 网络性能"
+msgid "Using BBR2 CCA can improve TCP network performance effectively"
+msgstr "使用 BBR2 拥塞控制算法可以有效提升 TCP 网络性能"
 
 msgid "FullCone NAT"
 msgstr "全锥形 NAT"
diff -Naur a/root/etc/init.d/turboacc b/root/etc/init.d/turboacc
--- a/root/etc/init.d/turboacc	2022-03-12 17:46:47.410005861 +0000
+++ b/root/etc/init.d/turboacc	2022-03-12 17:47:13.552215542 +0000
@@ -24,7 +24,7 @@
 
 	[ ! -e "/lib/modules/$(uname -r)/nft_flow_offload.ko" ] && { sw_flow="0"; hw_flow="0"; }
 	[ ! -e "/lib/modules/$(uname -r)/fast-classifier.ko" ] && { sfe_flow="0"; sfe_bridge="0"; sfe_ipv6="0"; }
-	[ ! -e "/lib/modules/$(uname -r)/tcp_bbr.ko" ] && bbr_cca="0"
+	[ ! -e "/lib/modules/$(uname -r)/tcp_bbr2.ko" ] && bbr_cca="0"
 	[ ! -e "/lib/modules/$(uname -r)/nft_fullcone.ko" ] && fullcone_nat="0"
 }
 
@@ -247,7 +247,7 @@
 	}
 
 	if [ "${bbr_cca}" -eq "1" ];  then
-		sysctl -w net.ipv4.tcp_congestion_control="bbr"
+		sysctl -w net.ipv4.tcp_congestion_control="bbr2"
 	else
 		sysctl -w net.ipv4.tcp_congestion_control="cubic"
 	fi
@@ -337,7 +337,7 @@
 			exit 0 || exit 1
 		;;
 	"bbr")
-		[ "x$(cat "/proc/sys/net/ipv4/tcp_congestion_control" 2>"/dev/null")" = "xbbr" ] && \
+		[ "x$(cat "/proc/sys/net/ipv4/tcp_congestion_control" 2>"/dev/null")" = "xbbr2" ] && \
 			exit 0 || exit 1
 		;;
 	"dns")
